lib/dom-helpers.mjs

```diff
@@ -0,0 +1,19 @@
+/**
+ * Creates an element with the given attributes
+ *
+ * TODO: Extend to allow passing props and children if necessary
+ * TODO: Make available to components once support for older browsers is dropped (add tests)
```

Comment:
Do we know for sure these are things we want to do, or are these more like ideas for the future? I'd be tempted to leave them out unless we know for sure.

Reply:
Probably more ideas at that stage, but I think there's ground for having a helper function for creating DOM elements with specific attributes and children quickly. That would make some of the code of the Accordion and CharacterCount less verbose I believe.

I'll drop the comments and move the discussion about this into an issue to avoid confusion.

Reply:
Issue created for discussion there: https://github.com/alphagov/govuk-frontend/issues/2894